apps/3000-home/next.config.js

```diff
@@ -89,6 +89,9 @@ const nextConfig = {
       checkout: `checkout@http://localhost:3002/_next/static/${
         isServer ? 'ssr' : 'chunks'
       }/remoteEntry.js`,
+      home_app: `home_app@http://localhost:3000/_next/static/${
```

Comment:
Why does it need to assert itself

Reply:
This is to verify that circular remote in single runtime chunk does not cause collision and overwrite chunkLoadingGlobal - its not needed, but kept to ensure E2E passes in scenarios where host uses own remote.